docs/validators/notation.md

```diff
@@ -0,0 +1,253 @@
+
+```bash
+cat <<EOF > ./trustpolicy.json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "my-images",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict"
+            },
+            "trustStores": [ "ca:mykey" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+EOF
+```
+
+Use `notation policy import` to import the policy:
+
+```bash
+notation policy import ./trustpolicy.json
+```
+
+### Verifying signatures
+
+You can verify signatures using the Notation CLI:
+
+```bash
+# Verify an image signature
+notation verify <IMAGE_URI>
+```
+
+### Configuring Connaisseur
+
+To use Connaisseur with Notation, configure a validator in `charts/connaisseur/values.yaml` with your signing certificate as a trust root.
+The entry in `.application.validators` should look like this:
+
+```yaml title="charts/connaisseur/values.yaml"
+- name: mynotation
+  type: notation
+  trustRoots:
+    - name: default
+      cert: |
+        -----BEGIN CERTIFICATE-----
+        MIIDrjCCApagAwIBAgIUfA/t/J6eINSu566aAozkOjQKey4wDQYJKoZIhvcNAQEL
+        BQAwbDELMAkGA1UEBhMCREUxDzANBgNVBAgMBkJlcmxpbjEPMA0GA1UEBwwGQmVy
+        # ... rest of your certificate ...
+        -----END CERTIFICATE-----
+```
+
+In `.application.policy`, add a pattern to match your repository:
+
+```yaml title="charts/connaisseur/values.yaml"
+- pattern: "docker.io/myrepo/*:*"  # YOUR REPOSITORY
+  validator: mynotation
+```
+
+After installation, Connaisseur will verify your images against the configured certificates:
+
+```bash
+helm install connaisseur helm --atomic --create-namespace --namespace connaisseur
+```
+
+### Testing validation
+
+You can test the validation by deploying pods with signed and unsigned images:
+
+```bash
+# This should succeed if the image is properly signed
+kubectl run signed-app --image=docker.io/myrepo/myimage:signed
+
+# This should fail if the image is not signed or signed with a different certificate
+kubectl run unsigned-app --image=docker.io/myrepo/myimage:unsigned
+```
+
+## Configuration options
+
+`.application.validators[*]` in `charts/connaisseur/values.yaml` supports the following keys for Notation (refer to [basics](../basics.md#validators) for more information on default keys):
+
+| Key | Default | Required | Description |
+| - | - | - | - |
+| `name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
+| `type` | - | :heavy_check_mark: | `notation`; the validator type must be set to `notation`. |
+| `trustRoots[*].name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
+| `trustRoots[*].cert` | - | :heavy_check_mark: | X.509 certificate in PEM format used for signature verification. This should be the root signing certificate or a trusted CA certificate. |
+| `trustRoots[*].tsCert` | - | - | X.509 certificate in PEM format used for timestamp counter signature verification. |
```

Comment:
Are these "regular" certs by default, like those in OS cert pools? Asking, because if you usually have to configure this (as ts verification defaults to always), I'd state that. If on the other hand, you don't I would also highlight that this is more for a case where this is a dedicated PKI by choice

Reply:
so are you asking whether those certs need to be installed into the OS, or wether they can just be added here and be use for some kind of verification?